clients/admin-ui/src/features/connector-templates/ConnectorTemplateUploadModal.tsx

```diff
@@ -49,12 +55,24 @@ const ConnectorTemplateUploadModal: React.FC<RequestModalProps> = ({
 
   const [registerConnectorTemplate, { isLoading }] =
     useRegisterConnectorTemplateMutation();
+  const { refetch: refetchConnectionTypes } = useGetAllConnectionTypesQuery(
+    {
+      search: "",
+    },
+    {
+      skip: false,
+    }
+  );
 
   const handleSubmit = async () => {
     if (uploadedFile) {
       try {
         await registerConnectorTemplate(uploadedFile).unwrap();
         toast(successToastParams("Connector template uploaded successfully."));
+
+        // refresh the connection types
+        const { data } = await refetchConnectionTypes();
+        dispatch(setConnectionOptions(data?.items ?? []));
```

Comment:
Refreshing connection options on success so we have an updated map of connector types to icons